scripts/radossim.py

```diff
@@ -80,7 +109,7 @@ def kvThread(env, srcQ, targetLat=5000, measInterval=100000):
                 batchReqSize += reqSize
         # Process batch
         kvQDispatch = env.now
-        yield env.timeout(latModel(batchReqSize))
+        yield env.timeout(latModel(batchReqSize, bm.bytesWritten, latency_model))
```

Comment:
Here it becomes clear that `latModel` should really be an instance of a class with its own state instead of a stateless function: This has the following advantages:

1. The model file can be loaded and parsed at initiation time
2. `bytesWritten` can be part of the `LatModel` instance instead of the `BatchManagement` instance.

Reply:
```suggestion
        yield env.timeout(latModel.getLatency(batchReqSize))
```

Reply:
It's done in LatencyModel class.